pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -38,10 +38,17 @@ const (
 
 //createNodeAllocatableCgroups creates Node Allocatable Cgroup when CgroupsPerQOS flag is specified as true
 func (cm *containerManagerImpl) createNodeAllocatableCgroups() error {
+	nodeAllocatable := cm.internalCapacity
```

Comment:
Side note:

This is essentially duplication of the code in https://github.com/kubernetes/kubernetes/pull/88198/files#diff-5dc6bd6afcca36d1b376b3c78accb0aaR69-R80. I do however think that it ok, since the alternative will require quite some refactoring (but feel free to disagree).